content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -317,6 +317,121 @@ Some values of an object are typically generated before the object is persisted.
+
+
+### Conflicts
+
+A conflict is a special status error that occurs when an `Apply` operation tries to change a field, which another user also claims to manage. This will prevent an applier from unintentionally overwriting the value set by another user. When this occurs, the applier has 3 options to resolve the conflicts:
+
+* **Overwrite value, become sole manager:** If overwriting the value was intentional (or if the applier is an automated process like a controller) the applier should set the `force` query parameter to true and make the request again. This will force the operation to succeed, change the value of the field, and remove the field from all other managers' entries in managedFields.
+* **Don't overwrite value, give up management claim:** If the applier doesn't care about the value of the field anymore, they can remove it from their config and make the request again. This will leave the value unchanged, and cause the field to be removed from the applier's entry in managedFields.
+* **Don't overwrite value, become shared manager:** If the applier still cares about the value of the field, but doesn't want to overwrite it, they can change the value of the field in their config to match the value of the object on the server, and make the request again. This will leave the value unchanged, and cause the field's management to be shared by the applier and all other field managers that already claimed to manage it.
+
+
+### Comparison with Client Side Apply
+
+A consequence of the conflict detection and resolution implemented by Server Side Apply is that an applier will always have up to date field values in their local state. If they don't, they will get a conflict the next time they apply. Any of the three options to resolve conflicts will result in the applied config being an up to date subset of the object on the server's fields.
+
+This is different from in Client Side Apply, where outdated values which have been overwritten by other users are left in an applier's local config. These values only become accurate when the user upates that specic field, if ever, and an applier has no way of knowing whether their next apply will overwrite other users' changes.
+
+Another difference is that an applier using Client Side Apply is unable to change the API version they are using, but Server Side Apply supports this use case.
+
+
+### Custom Resources
 
+Server Side Apply currently treats all custom resources as unstructured data. All keys are treated the same as struct fields, and all lists are considered atomic. In the future, it will use the validation field in Custom Resource Definitions to Custom Resource authors to definte how to how to merge their own objects.
```

Comment:
"In the future, it will use the validation field in Custom Resource Definitions to Custom Resource authors to definte how to how to merge their own objects."

I made typos here too... should say:

"In the future, it will use the validation field in Custom Resource Definitions to **allow** Custom Resource authors to **define** how to how to merge their own objects."